tools/cmake/mbed_greentea.cmake

```
@@ -70,6 +81,17 @@ macro(mbed_greentea_add_test)
 
     mbed_set_post_build(${MBED_GREENTEA_TEST_NAME})
 
+    add_test(
+        NAME ${MBED_GREENTEA_TEST_NAME}
+        COMMAND mbedhtrun
+            -m ${MBED_TARGET}
+            -f ${CMAKE_CURRENT_BINARY_DIR}/${MBED_GREENTEA_TEST_NAME}.bin
```

```suggestion
            -f ${MBED_GREENTEA_TEST_NAME}.bin
```
Forgot to mention: This can be simplified because `${CMAKE_CURRENT_BINARY_DIR}` is the default working directory: https://cmake.org/cmake/help/latest/prop_test/WORKING_DIRECTORY.html

---

Also, some targets have .hex images. We can use `${MBED_OUTPUT_EXT}` defined in `mbed_config.cmake`, and fall back to `bin` if it's not set or an empty string.